docs/ReleaseNotes.md

```diff
@@ -0,0 +1,45 @@
+# Release Notes
+This document contains information about the releases of this crate.
+
+## Version 0.1.0
+The Networking team is happy to announce the release of 0.1.0 of the [laminar crate](https://github.com/amethyst/laminar). 
+It provides UDP networking modified for the needs of game networking. 
+Most of the techniques used were published and detailed by [Glenn Fiedler](https://gafferongames.com/). 
+We’d like to extend a special thanks to him and his articles.
+
+**Features**
+- UDP-based protocol
+- Automatic Fragmentation
+- RTT estimation
+- Connection tracking.
+- Unreliable and Reliable sending of packets.
+- Protocol version monitoring.
+- A link conditioner to simulate packet loss and latency
+- Good error handling with **zero** panics.
+- Well tested by integration and unit tests
+- Benchmarks
+
+**Workflow**
+
+As a team, we adopted the following the workflow agreements. When we begin work on the amethyst_network crate, we’ll use the same agreements. They are focused on maintaining a high level of quality in the code, and for working with a highly distributed team. We’re including them here as some of the other teams may find them of use.
+
+- All warnings produced by `cargo test` are treated as errors by the CI/CD system
+- All 'Clippy' warnings are treated as errors by the CI/CD system
+- We use 'Kcov' to track our code coverage; we do not have a required minimum, rather we use this as a potential indicator of issues
+- We included sample code about using the library
+- Setting up a benchmarking framework so we can track regressions
+- Unit and integration tests, as well as release testing with docker-compose
+
+**Style Guidelines**
```

Comment:
can we try and follow this https://github.com/amethyst/amethyst/blob/master/docs/CONTRIBUTING.md? style wise atleast.

so this section relates to this https://github.com/amethyst/amethyst/blob/master/docs/CONTRIBUTING.md#pull-requests